ibm/service/power/ibm_pi_constants.go

```diff
@@ -7,6 +7,7 @@ const (
 	Arg_CloudConnectionName                 = "pi_cloud_connection_name"
 	Arg_CloudInstanceID                     = "pi_cloud_instance_id"
 	Arg_DatacenterZone                      = "pi_datacenter_zone"
+	Arg_Description                         = "pi_description"
```

Comment:
This should be description
```suggestion
	Arg_Description                         = "description"
```